Gems/Terrain/Assets/Shaders/Terrain/TerrainSrg.azsli

```diff
@@ -50,7 +51,7 @@ ShaderResourceGroup TerrainSrg : SRG_Terrain
         uint m_heightImageIndex; // only first 16 bits used
 
         // 16 byte aligned
-        uint2 m_padding;
+        uint m_padding;
```

Comment:
A struct in hlsl is always going to be 16 byte aligned, so these aren't even technically necessary - their inclusion is to make the members match up with the c++ side to avoid any confusion.